styles/atom-ide-outline.less

```diff
@@ -42,7 +43,7 @@
       padding: 2px 5px;
       white-space: nowrap;
 
-      padding-left: ~"calc(var(--editor-tab-length) * var(--level-indent-size) * var(--indent-level) + 5px + @{fold-width})";
+      padding-left: ~"calc(var(--editor-tab-length) * var(--level-indent-size) * var(--indent-level) + @{fold-width})";
```

Comment:
These 5px come from `padding` definition two lines above. Since indentation is supposed to be "in addition" to any padding that already exists.

Reply:
I'm not particularly attached to those 5px, just thought I should explain where those came from.

Reply:
I added it back in 
https://github.com/atom-community/atom-ide-outline/pull/96/commits/360249bd623c832b393a4881cf90766c0f9c8a2f